test/visual/message-list.html

```diff
@@ -17,7 +17,9 @@
 <body>
   <div id="tests" style="padding: 10px">
     <vaadin-message-list
-      items='[{"text":"Hello list","time":"yesterday", "user": {"name":"Matt Mambo","abbr":"MM","colorIndex":"1"}},{"text":"Another message","time":"right now", "user": {"name":"Linsey Listy","abbr":"LL","colorIndex":"2","img":"/test/visual/avatars/avatar.jpg"}}]'>
+      items='[{"text":"Hello list","time":"yesterday", "userName":"Matt Mambo", "userAbbr":"MM",
+      "userColorIndex":1},{"text":"Another message","time":"right now", "userName":"Linsey Listy",
+      "userAbbr":"LL","userColorIndex":2, "userImg":"/test/visual/avatars/avatar.jpg"}]'>
```

Comment:
```suggestion
      items='[{"text":"Hello list","time":"yesterday","userName":"Matt Mambo","userAbbr":"MM",
      "userColorIndex":1},{"text":"Another message","time":"right now","userName":"Linsey Listy",
      "userAbbr":"LL","userColorIndex":2,"userImg":"/static/img/avatar.jpg"}]'>
```

Reply:
This change broke the tests. I was too happily just accepting all the URL changes, but the /test/ package should actually have the `/test/visual/avatars/` URL for img.

Reply:
Right, my bad.